language/documentation/book/src/coding-conventions.md

```diff
@@ -65,7 +65,7 @@ module 0x1::Importer {
 ## Comments
 
 - Each module, struct, and public function declaration should be commented
-- Move has both doc comments `///`, regular single-line comments `//`, and block comments `/* */`
+- Move has doc comments `///`, regular single-line comments `//`, block comments `/* */` and block doc comments `/** */`
```

Comment:
```suggestion
- Move has doc comments `///`, regular single-line comments `//`, block comments `/* */`, and block doc comments `/** */`
```

Reply:
There's already docs on documentation generation/doc comments in a [README](https://github.com/diem/diem/blob/main/language/move-prover/doc/user/docgen.md) but nothing in the book. 

Probably worthwhile adding an issue for adding documentation generation to the book -- shouldn't be hard since we can pull over a lot from the docgen README.